library/alloc/src/boxed.rs

```diff
@@ -1416,6 +1448,14 @@ impl<T, const N: usize> From<[T; N]> for Box<[T]> {
 impl<T, const N: usize> TryFrom<Box<[T]>> for Box<[T; N]> {
     type Error = Box<[T]>;
 
+    /// Attempts to convert a `Box<[T]>` into a `Box<[T; N]>`.
+    ///
+    /// The conversion occurs in-place and does not require a
+    /// new memory allocation.
+    ///
+    /// # Errors
+    ///
+    /// Returns an error if `boxed_slice.len()`  does not equal `N`.
```

Comment:
It doesn't return an error, but the old `Box<[T]>`. There's also a double space after "`boxed_slice.len()`".

```suggestion
    /// Returns the old `Box<[T]>` in the `Err` variant if `boxed_slice.len()` does not equal `N`.
```